docs/bigquery-cdcTarget.md

```diff
@@ -40,6 +40,11 @@ Properties
 **Project ID**: Project of the BigQuery dataset. When running on a Dataproc cluster, this can be left blank,
 which will use the project of the cluster.
 
+**Location**: The location where the BigQuery dataset and GCS staging gcs bucket will get created. For example, 'us-east1' 
```

Comment:
Can omit 'gcs'. It can just be 'GCS staging bucket'

Reply:
done :)